contracts/beyondFramework/dca/libraries/DCAHistoryLib.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+library DCAHistoryLib {
+    struct HistoricalGauge {
+        uint256 amountSpent;
+        uint256 amountExchanged;
+    }
+
+    struct DCAHistory {
+        HistoricalGauge[] gauges;
+        uint256 current;
+    }
+
+    function addHistoricalGauge(
```

Comment:
HistoricalGauge and Gauge are mixed. Only here is HistoricalGauge but all others are Gauge. IMHO, sticking to a consistent naming gives us better readability.